src/components/SearchButton/index.tsx

```diff
@@ -0,0 +1,15 @@
+import * as S from "./styles";
+
+import { Search as SearchIcon } from "@styled-icons/octicons/Search";
+
+type SearchButtonProps = {
+  onClick: (ev: React.MouseEvent<HTMLButtonElement>) => void;
+};
+const SearchButton = ({ onClick }: SearchButtonProps) => (
+  <S.Wrapper onClick={onClick}>
+    <SearchIcon />
+    <span>Search</span>
+  </S.Wrapper>
+);
```

Comment:
This component does not match the layout, icon is different, spacing is disproportionate and the text is not the same.

![Captura de Tela 2021-04-24 às 01 19 00](https://user-images.githubusercontent.com/17678567/115946968-0e325d80-a49b-11eb-9e06-162769c934d3.png)
![Captura de Tela 2021-04-24 às 01 19 10](https://user-images.githubusercontent.com/17678567/115946975-14c0d500-a49b-11eb-9f90-2034651ab912.png)